decisions/0003-github-labels-decluttering.md

```diff
@@ -0,0 +1,85 @@
+# Use of Labels in OSCAL Project(s)
+
+Date: 02/17/2023
+
+## Status
+
+Approved
+
+## Context
+
+We wish to improve communication within the team and among the community members. Labels we use in OSCAL repositories can be a part of the effort.  We have discussed a number of options around labels and we will use an ADR to capture and document our approach moving forward.
+
+- Related to Issue [#1496](https://github.com/usnistgov/OSCAL/issues/1496)
+
+## Decision
+
+Existing labels were reviewed, a subset of labels is kept and new ones are defined.
+
+The lists of labels that follow will use across OSCAL repositories.
+
+#### General labels:
+
+- breaking change
+- community feedback needed
+- developer experience
+- discussion needed
+- epic
```

Comment:
Apologies, it was `faq` for FAQ, Frequently Asked Questions, I remember now. It was so we can use these labels in GitHub Discussions for the project and quickly filter on discussion posts that are candidates for FAQ status. https://github.com/usnistgov/OSCAL/discussions/1672

```suggestion
- epic
- faq
```